UNRELEASED.md

```diff
@@ -1,6 +1,11 @@
 # Unreleased
 
 > Place your changes below this line.
+
+**Added:**
+
+- Added Android bridge for `react-native-bpk-component-rating`.
```

Comment:
Should this have a more consumer-friendly description? I'm thinking that the bridging aspect should be invisible to consumers so we should just introduce it like any new component.

Reply:
Yeah makes sense, I'll update